cpp/include/cuspatial/soa_readers.hpp

```diff
@@ -16,63 +16,63 @@
 
 #pragma once
 
-typedef struct gdf_column_ gdf_column; // forward declaration
-
 namespace cuspatial {
 
```

Comment:
Maybe add an `experimental` namespace or similar

Reply:
Just because these APIs are fairly demo specific?

Reply:
Or because they are in development?

Reply:
Just to avoid name collisions with the legacy APIs of the same name. After removing the legacy stuff, then you can remove the `experimental` namespace, as in [cudf/4051](https://github.com/rapidsai/cudf/issues/4051). Alternatively, I suppose you could put the legacy APIs inside a `legacy` namespace to make it easier to remove them.